JSDemos/Demos/TextArea/Overview/Angular/app/app.component.css

```diff
@@ -3,10 +3,14 @@
   margin-top: 30px;
 }
 
-::ng-deep .full-width-content > .dx-widget {
+::ng-deep .full-width-content .dx-widget {
```

Comment:
It seems that this change is redundant and affects the final display

<img width="473" alt="image" src="https://github.com/DevExpress/devextreme-demos/assets/15689080/984d222a-1d06-4466-a172-2efc8cdaa4a1">

Please, fix it